src/components/WarningAnimateMessage/index.js

```diff
@@ -56,7 +56,7 @@ const WarningAnimatedMessage = ({isSHow, top = 100, left = 110}) => {
 
   React.useEffect(() => {
     getSpecificCache(OPEN_FIRST_TIME, (cache) => {
-      if (!cache) setIsRunAnimated(isSHow);
+      if (!cache?.isOpen) setIsRunAnimated(isSHow);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is a good one. It adds an extra check to ensure that `cache` is not null or undefined before trying to access the `isOpen` property, which can prevent potential runtime errors.
```diff
- if (!cache) setIsRunAnimated(isSHow);
+ if (!cache?.isOpen) setIsRunAnimated(isSHow);
```
However, it seems like there might be a logical error. The condition `!cache?.isOpen` will be true when `cache` is `null`, `undefined`, or when `isOpen` is `false`. If the intention was to run the animation only when `cache` exists and `isOpen` is `false`, then the condition should be `cache && !cache.isOpen`.

Here's the corrected code:
```diff
- if (!cache?.isOpen) setIsRunAnimated(isSHow);
+ if (cache && !cache.isOpen) setIsRunAnimated(isSHow);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->